package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/muse-components",
-  "version": "1.3.91",
+  "version": "1.3.92",
```

The release process should take care of bumping the version. No need to do it in this PR
```suggestion
  "version": "1.3.91",
```

---

Yeah, I just fixed this 😆  This was leftover from when I was trying to fix the tests.